### PR TITLE
Appended Beta Label to MS Teams documentation

### PR DESCRIPTION
**Pull request by cwarnermm**

### CODE REVIEW

File: source/about/embed-mattermost-app-within-microsoft-teams.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Embed the Mattermost App within Microsoft Teams
-================================================
+Embed the Mattermost App within Microsoft Teams (Beta)
```
Comment (johndavidlugtu) [inline source/about/embed-mattermost-app-within-microsoft-teams.rst:1]: Experimental

Comment (cwarnermm) [inline source/about/embed-mattermost-app-within-microsoft-teams.rst:1]:
```suggestion
Embed the Mattermost App within Microsoft Teams (Experimental)
```

File: source/about/embed-mattermost-app-within-microsoft-teams.rst [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+======================================================
```
Comment (cwarnermm) [inline source/about/embed-mattermost-app-within-microsoft-teams.rst:2]:
```suggestion
==============================================================
```

**Review by johndavidlugtu — APPROVED** [review]
Thanks Carrie

**Pull request merged**